dask/array/stats.py

```diff
@@ -263,8 +263,11 @@ def kurtosis(a, axis=0, fisher=True, bias=True, nan_policy="propagate"):
     if fisher:
         return vals - 3
     else:
+        if vals.ndim == 0:
+            # TODO: scalar, min is a workaround
+            return vals.min()
```

Comment:
Is this a common workaround? Why not use `.item` like the previous note suggests?

Reply:
There is no `.item()` in this case. I think it's still one of those methods that haven't been implemented in dask.

Reply:
Ah right right. This is fine for me.